include/tester/ssp/SSPAccount.hpp

```diff
@@ -47,15 +47,52 @@ class SSPAccount : public pj::Account
 
     void onIncomingCall(pj::OnIncomingCallParam &iprm) override
     {
-        pj::Call *call = new SSPCall(*this, iprm.callId);
-        pj::CallInfo ci = call->getInfo();
-        pj::CallOpParam prm;
-        
-        std::cout << "*** Incoming Call: " <<  ci.remoteUri << " ["
-                  << ci.stateText << "]" << std::endl;
-        
-        _calls.push_back(call);
-        prm.statusCode = (pjsip_status_code)200;
-        call->answer(prm);
+        if(!hasActiveCall())
+        {
+            _call = new SSPCall(*this, iprm.callId);
+            pj::CallInfo ci = _call->getInfo();
+            pj::CallOpParam prm;
+            
+            std::cout << "*** Incoming Call: " <<  ci.remoteUri << " ["
+                    << ci.stateText << "]" << std::endl;
+            
+            prm.statusCode = (pjsip_status_code)200;
+            _call->answer(prm);
+        }
     }
+
+    void removeCall()
+    {
+        delete _call;
+    }
+    
+    bool hasActiveCall()
+    {
+        pj::CallInfo callInfo = _call->getInfo();
+        if(callInfo.state == PJSIP_INV_STATE_CONFIRMED)
```

Comment:
you can return the statement rather than check if it is true or false.

Reply:
so I don't need that function at all, because I can only check the statement

Reply:
If you only check this condition once, there's no need for this function. however, if you repeat this condition elsewhere, a function is more comfortable to use.

Reply:
I renew this function and keep it for now maybe later i will remove it